components/raftstore/src/store/memory.rs

```diff
@@ -51,9 +51,12 @@ pub fn needs_evict_entry_cache() -> bool {
     if memory_usage_reaches_high_water() {
         let usage = get_global_memory_usage();
         let ec_usage = MEMTRACE_ENTRY_CACHE.sum() as u64;
-        // Evict cache if entry cache memory usage reaches 1/3 of global,
-        // or 1GiB for small instances.
-        if ec_usage > GIB && (ec_usage > usage / 3 || usage <= 3 * GIB) {
+        // Evict if entry cache memory usage reaches 1/5 of global, or 1GiB for small instances.
+        // So for different system memory capacity, cache evict happens:
+        // * system=8G,  memory_usage_limit=6G,  evict=1.2G
+        // * system=16G, memory_usage_limit=12G, evict=2.4G
+        // * system=32G, memory_usage_limit=24G, evict=4.8G
+        if ec_usage > GIB && ec_usage > usage / 5 {
```

Comment:
How about the total memory is 128GB? I prefer 'if ec_usage > GIB && (ec_usage > usage/5 || ec_usage >= 5GB)'